### ISSUE TITLE
Add events to dashboards

### ISSUE THREAD

**Issue by dragonsahead** [metabase/metabase#22630]
**Is your feature request related to a problem? Please describe.**
Events are a really cool feature when you want to try to associate things that happened to the actual data, but this is right now limited to the question level. If you want to show this on a dashboard, this is not possible right now.

**Describe the solution you'd like**
Render in the questions in dashboards

**Describe alternatives you've considered**
NA

**How important is this feature to you?**
Requested by a customer

**Additional context**
NA

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by stevazi** [comment]
any update on this?

**Comment by levon-sib** [comment]
Any updates?

**Comment by doscrayon** [comment]
Is this already on roadmap?

**Comment by 08351ty** [comment]
Any updates on this? Would LOVE to have events showable on dashboards instead of just in questions. If there's any workaround pls share as well

**Comment by AndrewLWilson** [comment]
Events on questions has really been a game changer.  Getting them on dashboards would be amazing!!!

**Comment by lucas-loreto** [comment]
Keeping track of major events was a game changer. Great to see impacts of releases and major bugs. Please consider upgrading it, there's so much value to unlock!
1. Events visible on dashboards
2. Events on a collection automatically visible on its sub-collections

**Comment by PeerRich** [comment]
we'd love to use this over at https://cal.com/open

**Comment by Vanomis** [comment]
Any update guys ? 🙏 Looking forward to use it!

**Comment by JonathanHindi** [comment]
This would be amazing. Are there any plans to include it in the roadmap?

**Comment by StreamlinedStartup** [comment]
Would love to see events in dashboard view, particularly if it is enabled on a question.

**Comment by dayananabieva** [comment]
yes, plz - it would be really cool feature, i was surprised when didn't find how to add events on dashboards

**Comment by jae-ryu** [comment]
upvoting as well - we would love to see this on the dashboards themselves, not just on the questions.

**Comment by samanthawobst1** [comment]
Please! 🙏

**Comment by timbo-tj** [comment]
100%, this would be awesome.

**Comment by spellfusion** [comment]
Bump

**Comment by aruizhong** [comment]
100%, this would be awesome.

**Comment by farhanjo** [comment]
Could someone from the MB team help us understand what's like the average number of thumbs up on an issue that usually goes into development? 
This way, it would be nice to get an idea of how far off a feature is potentially from being prioritized

**Comment by gonfsoto** [comment]
Much needed feature to take full potential of events

**Comment by Timothyleeclark** [comment]
I agree with this, its nice to have a timeline with annotations, but would of to be able to display a table with the events and the dates for dashboard purpose.

**Comment by shyrahnama** [comment]
pretty pleeeeeease

**Comment by joakimwar** [comment]
Yes please

**Comment by acorso522** [comment]
Please.

**Comment by matclayton** [comment]
Pretty pretty please with sugar on the top

**Comment by artemisss** [comment]
Add please this feature

**Comment by ddobrinskiy** [comment]
please please please, +1 to this

**Comment by the21st** [comment]
how is this not yet shipped after 4 years

**Comment by Eirenliel** [comment]
how is this not a thing still 😭 makes the whole system kind of useless...

**Comment by MonikaRiehn** [comment]
Please add this feature
